Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -253,9 +268,522 @@ def fetch_incidents(client):
+    # correct offset:limit after the api call.
+    args["limit"] = offset + args["limit"]
+
+    host_forensics = get_api_filtered_response(
+        client=client, url_suffix=f"/profiles/host/{host_id}/forensic", offset=offset, limit=limit, args=args
+    )
+
+    if host_forensics:
+        context_output = {
+            "HostID": host_id,
+            "Forensics": host_forensics
+        }
+
+        return context_output, tableToMarkdown(
+            name="Host forensics report", t=host_forensics, headers=["type", "app", "path", "command"], removeNull=True
+        )
+    return {}, tableToMarkdown(name="Host forensics report", t=[])
+
+
+@validate_limit_and_offset
+def get_profile_host_forensic_list(client: PrismaCloudComputeClient, args: dict) -> CommandResults:
+    """
+    Returns runtime forensics data for a specific host.
+    Implement the command 'prisma-cloud-compute-host-forensic-list'
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+        args (dict): prisma-cloud-compute-host-forensic-list command arguments.
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    host_id = args.pop("id")
+    context, table = build_host_forensic_response(client=client, host_id=host_id, args=args)
+
+    return CommandResults(
+        outputs_prefix='prismaCloudCompute.hostForensic',
+        outputs=context if context else None,
+        readable_output=table,
+        outputs_key_field="HostID"
+    )
+
+
+def get_console_version(client: PrismaCloudComputeClient) -> CommandResults:
+    """
+    Returns the version of the prisma cloud compute console.
+    Implement the command 'prisma-cloud-compute-console-version-info'.
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    version = perform_api_request(client=client, url_suffix="/version")
+
+    return CommandResults(
+        outputs_prefix="prismaCloudCompute.console.version",
+        outputs=version,
+        readable_output=tableToMarkdown(name="Console version", t={"version": version}, headers=["version"])
+    )
+
+
+def get_custom_feeds_ip_list(client: PrismaCloudComputeClient) -> CommandResults:
+    """
+    Get all the BlackListed IP addresses in the system.
+    Implement the command 'prisma-cloud-compute-custom-feeds-ip-list'
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    feeds = perform_api_request(client=client, url_suffix="/feeds/custom/ips")
+
+    if feeds and "modified" in feeds:
+        feeds["modified"] = parse_date_string_format(date_string=feeds["modified"])
+
+    return CommandResults(
+        outputs_prefix="prismaCloudCompute.customFeedIP",
+        outputs=feeds,
```

Comment:
unfortunately I am not sure about that, in the test env it is empty, according to the documentation it can be returned as part of the response (i guess not only empty values, otherwise it would simply be useless).  
Should I remove it from context?

Reply:
FYI, just spoke with twistlock PM, this is not required and they will remove it from the api, so ill remove it as well from our responses to the context